tutorials/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -35,7 +35,7 @@ Here is a complete class example based on these guidelines:
 
     signal state_changed(previous, new)
 
-    export var initial_state = NodePath()
+    @export var initial_state := NodePath()
     var is_active = true setget set_is_active
```

Comment:
This is another `setget` example that we need to update. I think we can use a method reference here.